### PR TITLE
why did i spend my time on this

### PR DESCRIPTION
**Pull request by hornygranny**

### CODE REVIEW

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1334,14 +1334,14 @@
 			usr << "This cannot be used on instances of type /mob/living/silicon/ai"
 			return
 
-		if(alert(usr, "Send [key_name(M)] to Prison?", "Message", "Yes", "No") != "Yes")
+		if(alert(usr, "Send [key_name(M)] to the Cuck Sheds??", "Message", "Yes", "No") != "Yes")
```
Comment (MrPerson) [inline code/modules/admin/topic.dm]: Let's all agree this double question mark is intentional

**Comment by Razharas** [comment]
/pol/station 13

**Comment by Cheridan** [comment]
this is a fix, obviously

**Comment by KorPhaeron** [comment]
This is a christian server though

**Comment by optimumtact** [comment]
Reopened, stop issue warring with the head coder

**Comment by KorPhaeron** [comment]
It doesn't fit my design vision though

**Comment by optimumtact** [comment]
oh I see

**Pull request closed without merging**